requirements.txt

```diff
@@ -13,6 +13,7 @@ git+https://github.com/Aalto-LeTech/django-colortag.git@2.1.1#egg=django-colorta
 feedparser~=5.2.1
 html5lib
 lxml ~= 4.3.2
+#lz4~=3.1.0
```

Comment:
Spaces make these a bit more readable (see lxml above) and make it more clear that it's not exacpt match. Apparently many others don't have space either (which is probably the reason I hadn't noitced those before). Sadly, I didn't find any discussion about this in the internet.

I'll mark this to be added to the style guide too.

Reply:
Oops, it should be == and not ~=, is spaces used in that case also?

Reply:
The `~=` is pretty good in our kind of use. Requirements file is use more for requirements and less as a lock (see package-lock.json with JavaScript npm and Gemfile.lock with Ruby bundler). Because it's a comment, both version requirements are fine.

Most of the requirements have `==` and no spaces, as the file has been created with `pip freeze`, I assume.